assets/datatables/tables/crops_dt.tres

```diff
@@ -21,7 +21,6 @@ point_count = 5
 
 [sub_resource type="Resource" id="Resource_86ah0"]
 script = ExtResource("1_kus3k")
-attribute = 0
```

Comment:
What's going on with these values being removed? is it a Godot serialisation thing to remove 0 values? we should be careful this doesn't remove data from the tables